content/master/concepts/pods.md

```diff
@@ -0,0 +1,321 @@
+status of deployed resources and correcting any "drift." After checking a 
+resource Crossplane waits some time and checks again.
+
+{{<hint "note" >}}
+The Crossplane pod only reconciles core Crossplane components, including claims
+and composite resources. Providers are responsible for reconciling their managed
+resources. 
+{{< /hint >}}
+
+Crossplane monitors resources in one of two ways:
+1. A Kubernetes _[watch](https://kubernetes.io/docs/reference/using-api/api-concepts/#efficient-detection-of-changes)_
+2. Periodic polling
+
+Crossplane requests that Kubernetes notifies Crossplane of any changes on
+objects. This notification tool is a _watch_. 
+
+Watched objects include providers, managed resources and composite resource 
+definitions.
+
+For objects that Kubernetes can't provide a watch for, Crossplane
+periodically poll the resource to find it's state. The default polling rate is
+one minute. Change the polling rate with the `---poll-interval` pod argument.
+
+Reducing the poll-interval value causes Crossplane to poll resources more
+frequently. This increases the load of the Crossplane pod and
+results in more frequent provider API calls.
+
+<!-- vale write-good.TooWordy = NO -->
+<!-- allow "maximum" -->
+Increasing the poll-interval causes Crossplane to poll resources less
+frequently. This increases the maximum time until Crossplane
+discovers changes in the cloud provider that require updating. 
+<!-- vale write-good.TooWordy = YES -->
+
+Managed resources use polling.
+
+{{< hint "note" >}}
+Managed resources watch for Kubernetes events like deletion or changes to
+their `spec`. Managed resources rely on polling to detect changes in the
+provider environment.
+{{< /hint >}}
+
+Crossplane checks all resources, both watched and polled resources, to
+confirm they're in the desired state. Crossplane does this every one hour by
+default. Use the `--sync-interval` Crossplane pod argument to change this
+interval. 
+
+The `--max-reconcile-rate` rate defines two different settings at the same time:
+
+* The global retry rate for checking failed objects.
+* The number of threads created to reconcile resources.
+
+Reducing the `--max-reconcile-rate`, or making it smaller, increases the
+CPU resources Crossplane uses.
+
+Increasing the `--max-reconcile-rate`, or making it larger, reduces CPU 
+resources Crossplane uses, but increases the amount of time until changed 
+resources are fully synced. 
+
+{{< hint "important" >}}
+Most providers use their own `--max-reconcile-rate`. This determines the
+same settings for Providers and their managed resources. 
+{{< /hint >}}
+##### Reconcile retry rate
+
+When Crossplane checks a resource and it's not in the desired
+state they 100 times per second. If the resource is still not in the desired 
+state Crossplane checks the `--max-reconcile-rate` times per second. By default,
+the `--max-reconcile-rate` value of 10 means checking 10 times per second. 
```

Comment:
Re-opening this thread as there's still some strange grammar in here, e.g.:

> When Crossplane checks a resource and it's not in the desired state __they__ 100 times per second.

Does my comment above help communicate what `--max-reconcile-rate` does? I'd like to make sure we remove any mention of anything happening 100 times per second - it's wrong and I think it would be worrying to readers. Let me know if I can help better explain how this works. Happy to take a stab at writing this section myself if necessary.

Reply:
I've simplified the wording here and removed the reference to 100x